src/components/CostEstimationContent/CostOperation.tsx

```diff
@@ -0,0 +1,191 @@
+            },
+          ],
+          description: `*Estimations based on the current gas average at ${Math.ceil(gwei)} gwei (ETH), ETH price at USD
+          $${price} for Ethereum and ${Math.ceil(maticGwei)} gwei (MATIC), MATIC price at USD $${maticPrice} for Polygon
+         as at ${dateTime}.`,
+        },
+      },
+      {
+        jobTitle: "The Importer",
+        icon: "/static/images/home/howItWorks/persona/persona3.png",
+        costInformation: {
+          costTitle: "Cost for The Exporter",
+          contractCost: [
+            {
+              contractIcon: "/static/images/cost/importer/surrender-ebl-icon.png",
+              contractTitle: "Surrender eBL",
+              contractPriceInEthereum: `${surrenderDocGas * priceFactor}`,
+              contractPriceInPolygon: `${surrenderDocGas * maticPriceFactor}`,
+            },
+          ],
+          description: `*Estimations based on the current gas average at ${Math.ceil(gwei)} gwei (ETH), ETH price at USD
+          $${price} for Ethereum and ${Math.ceil(maticGwei)} gwei (MATIC), MATIC price at USD $${maticPrice} for Polygon
+         as at ${dateTime}.`,
+        },
+      },
+      {
+        jobTitle: "The Banker",
+        icon: "/static/images/home/howItWorks/persona/persona4.png",
+        costInformation: {
+          costTitle: "Cost for The Exporter",
+          contractCost: [
+            {
+              contractIcon: "/static/images/cost/exporter/transfer-holdership-icon.png",
+              contractTitle: "Cost to transfer holdership",
+              contractPriceInEthereum: `${transferHoldershipGas * priceFactor}`,
+              contractPriceInPolygon: `${transferHoldershipGas * maticPriceFactor}`,
+            },
+            {
+              contractIcon: "/static/images/cost/exporter/transfer-ownership-icon.png",
+              contractTitle: "Cost to transfer ownership",
+              contractPriceInEthereum: `${transferOwnershipGas * priceFactor}`,
+              contractPriceInPolygon: `${transferOwnershipGas * maticPriceFactor}`,
+            },
+            {
+              contractIcon: "/static/images/cost/total-cost-icon.png",
+              contractTitle: "Total Cost",
+              contractPriceInEthereum: `${transferOwnershipGas * priceFactor + transferHoldershipGas * priceFactor}`,
+              contractPriceInPolygon: `${
+                transferOwnershipGas * maticPriceFactor + transferHoldershipGas * maticPriceFactor
+              }`,
+            },
+          ],
+          description: `*Estimations based on the current gas average at ${Math.ceil(gwei)} gwei (ETH), ETH price at USD
+          $${price} for Ethereum and ${Math.ceil(maticGwei)} gwei (MATIC), MATIC price at USD $${maticPrice} for Polygon
+         as at ${dateTime}.`,
+        },
+      },
+    ];
+  }, [dateTime, gwei, maticGwei, maticPrice, maticPriceFactor, price, priceFactor]);
```

Comment:
why do you like to put all the data bunched up like that?

anyway to me, this effect should need to depend on so many variables
as maticPxs, price, priceFactor, dateTime all depend on gwei/maticGwei only.

Reply:
ok